### ISSUE TITLE
Show calendar week instead of month below each "week column"

### ISSUE THREAD

**Issue by frdmn** [JustinFincher/GitHubContributionsiOS#1]
Right now it says, "`A A A A S S S S O O O O [...]`"

What do you think?

**Comment by KevinGAleman** [comment]
I like this idea, but with the approach used by Github on user profile pages, where they list the first three letters of the month next to the first full week of the month. I'll code it now and submit a PR to see if Justin agrees. :)

**Comment by JustinFincher** [comment]
@frdmn Well the widget can't afford two-digit numbers (eg "52") in such small space, so I just made it showing the first letter of the month. Surely it has some identification difficulty to know what week it exactly, but I think it's enough except June and July ("J" × 8).

@KevinGAleman I had the same idea when I am coding it, but later I found it seems unbalanced when showing in today center. The GitHub profile page has plenty free space to ease the unaligned feeling of month labels, but in a strict layout like iOS today center this feeling will pop-out and make it seems unbalanced. As for the PR, thank you for pointing out my typo (sorry for my limited English), but I will probably not merge it because of my personal view of the UX. 

Thank you all for paying attention to this repo :-).
